File: fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/objects/event/EventCategoryTest.java
package com.github.dedis.popstellar.model.objects.event;

import org.junit.Test;

import static org.hamcrest.CoreMatchers.is;
import static org.hamcrest.MatcherAssert.assertThat;

public class EventCategoryTest {

  @Test
  public void toStringTest() {
    assertThat(EventCategory.PAST.toString(), is("Past Events"));
    assertThat(EventCategory.PRESENT.toString(), is("Present Events"));
    assertThat(EventCategory.FUTURE.toString(), is("Future Events"));
  }
}

File: fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/model/objects/event/EventCategoryTest.kt
package com.github.dedis.popstellar.model.objects.event

import org.hamcrest.CoreMatchers
import org.hamcrest.MatcherAssert
import org.junit.Test

class EventCategoryTest {
  @Test
  fun toStringTest() {
    MatcherAssert.assertThat(EventCategory.PAST.toString(), CoreMatchers.`is`("Past Events"))
    MatcherAssert.assertThat(EventCategory.PRESENT.toString(), CoreMatchers.`is`("Present Events"))
    MatcherAssert.assertThat(EventCategory.FUTURE.toString(), CoreMatchers.`is`("Future Events"))
  }
}